SDNExpress/scripts/SDNExpress.ps1

```diff
@@ -2401,6 +2401,215 @@ Configuration ConfigureHostAgent
     }
 }
 
+Configuration ConfigureIDns
+{
+    
+	Import-DscResource -ModuleName 'PSDesiredStateConfiguration'
+
+	Node $AllNodes.Where{$_.Role -eq "NetworkController"}.NodeName

```

Comment:
This doesn't seem right - this means that you will execute these REST PUTs against every controller VM. I assume that you just want to execute a REST PUT once against the controller? If so, you should use: